lib/screenplay_web/controllers/pa_messages_api_controller.ex

```diff
@@ -1,6 +1,10 @@
 defmodule ScreenplayWeb.PaMessagesApiController do
   use ScreenplayWeb, :controller
 
+  require Logger
+
```

Comment:
**quibble:** doesn't look like you're using the Logger in here. can we remove this?

```suggestion
```